packages/browser/src/index.ts

```diff
@@ -77,4 +112,10 @@ export default class LogtoClient {
 
     return this.oidcConfig;
   }
+
+  protected clearSignInSession() {
+    sessionStorage.removeItem(this.logtoSignInSessionKey);
+  }
```

Comment:
why do we need this?

Reply:
after sign-in `handleCallback`, should clear sign-in session.

Reply:
Updated. PTAL

```ts
  protected get signInSession(): Optional<LogtoSignInSessionItem> {
    const jsonItem = sessionStorage.getItem(this.logtoSignInSessionKey);

    if (!jsonItem) {
      return undefined;
    }

    try {
      const item: unknown = JSON.parse(jsonItem);
      assert(item, LogtoSignInSessionItemSchema);

      return item;
    } catch (error: unknown) {
      throw new LogtoClientError('sign_in_session.invalid', error);
    }
  }

  protected set signInSession(logtoSignInSessionItem: Optional<LogtoSignInSessionItem>) {
    if (!logtoSignInSessionItem) {
      sessionStorage.removeItem(this.logtoSignInSessionKey);

      return;
    }

    const jsonItem = JSON.stringify(logtoSignInSessionItem);
    sessionStorage.setItem(this.logtoSignInSessionKey, jsonItem);
  }
```